dds_web/database/models.py

```diff
@@ -8,6 +8,7 @@
 import datetime
 
 # Installed
+import pytz
```

Comment:
Maybe I missed something, but is this module used in this file?

Reply:
No, it's not, will remove!